peer.go

```diff
@@ -996,6 +996,13 @@ out:
 				idleTimer.Reset(idleTimeout)
 				continue
 
+			// If the NodeAnnouncement has an invalid alias, then
+			// we'll log that error above and continue so we can
+			// continue to read messges from the peer.
+			case lnwire.ErrInvalidNodeAlias:
```

Comment:
This should have a test if possible. I think maybe this should be a pointer as well?

Reply:
There isn't a good way to make an accurate end to end test for this as we don't allow local nodes to set the value, and we don't yet have a p2p testing framework in place. Correct on pointer, just added! We do currently have unit tests that cover validation however.